packages/airnode-protocol/test/rrp/requesters/RrpBeaconServer.sol.js

```diff
@@ -65,7 +65,7 @@ beforeEach(async () => {
     .initializeAndGrantRoles(
       [managerRootRole, managerRootRole, managerRootRole, managerRootRole],
       [
-        Math.random(),
+        Math.random().toString(),
```

Comment:
After implementing the `bytes(description).length > 0` check I found out that these numbers were being sent as empty